cmake/tribits/examples/TribitsExampleProject/packages/wrap_external/CMakeLists.txt

```diff
@@ -62,31 +62,44 @@ endif()
 # start from scratch.
 
 #
-# B) Write the export makefile that will be used by the external project
+# B) Get the arguments for the configure of exteranl_func
```

Comment:
```suggestion
# B) Get the arguments for the configure of external_func
```

Reply:
Thanks! Addressed in TriBITSPub/TriBITS#430